config/main.py

```diff
@@ -929,6 +929,9 @@ def config(ctx):
 config.add_command(nat.nat)
 config.add_command(vlan.vlan)
 
+from .kube import kubernetes
```

Comment:
Can you move this import to the top of the file now? Are the cyclical dependencies fixed?

Reply:
yep. Moved.